src/core/qgsdataprovider.h

```diff
@@ -492,6 +493,30 @@ class CORE_EXPORT QgsDataProvider : public QObject
      */
     virtual bool renderInPreview( const QgsDataProvider::PreviewContext &context ); // SIP_SKIP
 
+    /**
+     * Retrieve collected Metadata from the Provider source
+     * \brief A structured metadata store for a map layer.
+     * \note
+     * \see QgsVectorLayer::setDataProvider
+     * \see QgsRasterLayer::setDataProvider
+     * \see setLayerMetadata
+     * \see mLayerMetadata
```

Comment:
Remove references to private member variables from public docs

Reply:
done